src/pip/_internal/commands/__init__.py

```diff
@@ -101,6 +98,7 @@ def create_command(name, **kwargs):
 
 
 def get_similar_commands(name):
+    # type: (str) -> Union[bool, Sequence[str]]
```

Comment:
IMHO the function should be tweaked a bit to return `Optional[str]` (or `Optional[Text]`).

Reply:
Hi @McSinyx 

Yes, by passing None instead of False. The usages at https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/help.py#L30 and https://github.com/pypa/pip/blob/master/src/pip/_internal/cli/main_parser.py#L87 shouldn't be affected as well by this.

Reply:
Should the usage be touched as well?  I mean `guess` could be false-y if the return value is an empty string.  I'm not sure if an empty guess should ever be considered as a good guess, but that's `difflib.get_close_matches`'s job.  Also how likely is its return value be an empty list?  Could we

    try:
        return close_commands[0]
    except IndexError:
        return None

Reply:
Hi @McSinyx 

Sure that's a good idea. Let me rename the `get_similar_commands` to `get_similar_command`